### PR TITLE
Database diagram

### PR DESCRIPTION
**Pull request by Uthmanbello**
## Highlights
- Created `schema_based_on_diagram.sql` file
- Many-to-many relationship was included
- FK indexes were also included
## Image
image

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+"date_of_birth" DATE NOT NULL
+);
+
+CREATE TABLE "treatments"(
+"id" SERIAL PRIMARY KEY,
+"type" VARCHAR(255) NOT NULL,
+"name" VARCHAR(255) NOT NULL,
+"medical_history_id" INTEGER NOT NULL REFERENCES "medical_histories"("id"),
+INDEX("medical_history_id")
+);
```
Comment (youmari) [inline schema_based_on_diagram.sql:31]: - [ ] There many too many relationships between `treatments` and `medical_histrories` as shown in  the diagram and in order to achieve that you need to add a table between them that will have the foreign keys  please make sure  they have indexes  as required

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @Uthmanbello , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @Uthmanbello, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Uthmanbello** [comment]
@anagudelogu, 
Thanks a lot

**Pull request merged**